client/src/services/sport-field.js

```diff
@@ -0,0 +1,20 @@
+import fetch from "isomorphic-fetch";
```

Comment:
Could you please explain what are you trying to do with this?

Reply:
Sorry, I tried in another branch mock the asynchronous petitions, now I fix it.